Mlem/Views/Tabs/Profile/User View.swift

```diff
@@ -69,34 +61,48 @@ struct UserView: View {
             progressView
         }
     }
+    
+    @ViewBuilder
+    private var moderatorButton: some View {
+        if let user = userDetails {
+            if !moderatedCommunities.isEmpty {
+                NavigationLink(value: UserModeratorLink(user: user, moderatedCommunities: moderatedCommunities)) {
+                    Image(systemName: "shield")
+                }
+            }
+        }
+    }
 
+    private func header(for userDetails: APIPersonView) -> some View {
+        CommunitySidebarHeader(
+            title: userDetails.person.displayName ?? userDetails.person.name,
+            subtitle: "@\(userDetails.person.name)@\(userDetails.person.actorId.host()!)",
+            avatarSubtext: $avatarSubtext,
+            avatarSubtextClicked: self.toggleCakeDayVisible,
+            bannerURL: shouldShowUserHeaders ? userDetails.person.banner : nil,
+            avatarUrl: userDetails.person.avatar,
+            label1: "\(userDetails.counts.commentCount) Comments",
+            label2: "\(userDetails.counts.postCount) Posts")
+    }
+    
     private func view(for userDetails: APIPersonView) -> some View {
         ScrollView {
-            CommunitySidebarHeader(
-                title: userDetails.person.displayName ?? userDetails.person.name,
-                subtitle: "@\(userDetails.person.name)@\(userDetails.person.actorId.host()!)",
-                avatarSubtext: $avatarSubtext,
-                avatarSubtextClicked: self.toggleCakeDayVisible,
-                bannerURL: shouldShowUserHeaders ? userDetails.person.banner : nil,
-                avatarUrl: userDetails.person.avatar,
-                label1: "\(userDetails.counts.commentCount) Comments",
-                label2: "\(userDetails.counts.postCount) Posts")
+            header(for: userDetails)
             
             if let bio = userDetails.person.bio {
                 MarkdownView(text: bio, isNsfw: false).padding()
             }
             
             Picker(selection: $selectionSection, label: Text("Profile Section")) {
-                Text(FeedType.overview.rawValue).tag(0)
-                Text(FeedType.comments.rawValue).tag(1)
-                Text(FeedType.posts.rawValue).tag(2)
+                Text("Overview").tag(0)
+                Text("Comments").tag(1)
+                Text("Posts").tag(2)
```

Comment:
This can be done with an `enum` using `allCases`--more concise in the Picker, automatically extensible (just add a case to the enum), more readable than the integer tags (see `VoteComplexStyle` and `SelectableSettingsItem`)